Tests/run_tests.py

```diff
@@ -128,7 +128,6 @@ def is_pypy():
         "Bio.PDB.Vector",
         "Bio.phenotype",
         "Bio.phenotype.parse",
-        "Bio.phenotype.phen_micro",
```

Comment:
Why are you removing this check? Don't the ``Bio.phenotype.phen_micro`` tests require NumPy?